elpis/engines/common/output/raw_to_elan.py

```diff
@@ -0,0 +1,13 @@
+import dict2xml
+import subprocess
+import re
+
+def convert_raw_to_elan(transcription_data, xml_path, elan_path, style_path="/elpis/elpis/engines/common/output/elan.xsl"):
```

Comment:
Needs typing and docstring.

Also, what is "raw" in this context?

Reply:
I call "raw" a basic raw file or string, without specified or obvious format.